.mergify.yml

```diff
@@ -1,3 +1,10 @@
+queue_rules:
```

Comment:
Can you clarify if the queue_rules for open source projects support multiple entries?

Reply:
Fixed.

Reply:
fixed